mono/utils/memfuncs.c

```diff
@@ -221,3 +234,57 @@ mono_gc_memmove_atomic (void *dest, const void *src, size_t size)
 	else
 		mono_gc_memmove_aligned (dest, src, size);
 }
+
+guint64
+mono_determine_physical_ram_size (void)
```

Comment:
Why “determinine” and not just “get”?

Reply:
I could get behind "get", but its an existing method and do I really want to rename in this change?